blocks/library/quote/index.js

```diff
@@ -7,7 +7,7 @@ import Editable from 'components/editable';
 
 const { parse, html, query } = hpq;
 
-const fromValueToParagraphs = ( value ) => value.map( ( paragraph ) => `<p>${ paragraph }</p>` ).join( '' );
+const fromValueToParagraphs = ( value ) => value ? value.map( ( paragraph ) => `<p>${ paragraph }</p>` ).join( '' ) : '';
```

Comment:
Yes this fixes the bug of inserting an empty quote block (using the inserter) since we're assuming the attributes are all nullable.